tests/test-cases/Group23-VIC-Machine-Service/23-02-VCH-List.robot

```diff
@@ -111,3 +111,17 @@ Get VCH List Within Invalid Datacenter and Compute Resource
 
     Verify Return Code
     Verify Status Not Found
+
+    [Teardown]   Run Secret VIC Machine Delete Command   %{VCH-NAME}
```

Comment:
For readability point of view, if this teardown is actually for preparing for the next test case of checking empty VCH list output, do you think if it's better if we move this cleanup and add it to the first step of the empty VCH list test?
@zjs

Reply:
will do